src/metal.cs

```diff
@@ -447,8 +449,7 @@ partial interface MTLComputeCommandEncoder : MTLCommandEncoder {
 		[Export ("waitForFence:")]
 		void Wait (IMTLFence fence);
 
-		[Mac (10,13, onlyOn64: true)]
-		[NoTV][NoiOS]
+		[Mac (10,13, onlyOn64: true), iOS (11,0), TV (11,0)]
```

Comment:
Minor but I try no to update existing availability attributes so I would have added `[iOS (11,0), TV (11,0)]` bellow the Mac one in replacement of `[NoTV][NoiOS]` (;